.github/workflows/clients-go.yml

```diff
@@ -52,34 +52,9 @@ jobs:
         run: ./zgo.bat test ./...
         working-directory: ./src/clients/go
 
-  build_sample:
```

Comment:
The build process for samples of all languages has been unified. They are now not only built but run. For Node, Go, and Java.